src/modules/filesystem_freeze.py

```diff
@@ -121,22 +118,24 @@ def run(self) -> Dict[str, Any]:
         :return: A dictionary containing the result of the test case.
         :rtype: Dict[str, Any]
         """
-        file_system = self._find_filesystem()
+        file_system, mount_point = self._find_filesystem()
 
-        self.log(
-            logging.INFO,
-            f"Found the filesystem mounted on /hana/shared: {file_system}",
-        )
-
-        if file_system:
-            read_only_output = self.execute_command_subprocess(FREEZE_FILESYSTEM(file_system))
+        if file_system and mount_point:
+            self.log(
+                logging.INFO,
+                f"Found the filesystem mounted on: {file_system} at {mount_point}",
+            )
+            read_only_output = self.execute_command_subprocess(
+                FREEZE_FILESYSTEM(file_system, mount_point)
+            )
             self.log(logging.INFO, read_only_output)
             self.result.update(
                 {
                     "changed": True,
-                    "message": "The file system (/hana/shared) was successfully mounted read-only.",
+                    "message": f"The file system ({mount_point}) was mounted read-only.",
```

Comment:
I assume the 'f' is a typo?

Reply:
No, it is a f-string (formatted string literals) in python.